src/operations/write.ts

```diff
@@ -178,6 +218,9 @@ const writeRoot = (ctx: Context, select: SelectionSet, data: Data) => {
     const fieldValue = data[fieldAlias];
 
     if (ctx.store.updates[fieldName]) {
+      // TODO: Should this really always replace the default logic?
```

Comment:
I suppose an updater should always have presedence over automatic behavior, `addTodo` --> the consumer of the graphcache decides how this should be written to the store. Neglecting the updater could lead to unexpected behavior

Reply:
I think it'll need to do both at all times. It's literally almost impossible to write a huge fragment for every mutation that has some extra updates 😅

Ultimately all entities will need to be written to the cache for consistency. It's the Query links that need manual updating. I expect that if a GraphQL API with best practices followed returns the data that was mutated by mutations, our cache will already pick up all the changed entities.

The updates job in such a case is just to convey how "Query" links were affected